test/scheduler_test.rb

```diff
@@ -313,7 +311,7 @@
     }
     assert_equal(
       { 'cron' => '* * * * *', 'class' => 'SomeIvarJob', 'args' => '/tmp/75' },
-      Resque.decode(Resque.redis.hget(:schedules, 'my_ivar_job'))
+      Resque.schedule['my_ivar_job']
```

Comment:
Yes the previous semantics assumed that redis is the single source of truth thus it looked in redis for schedules